packages/cozy-sharing/src/components/ShareDialogCozyToCozy.jsx

```diff
@@ -32,76 +159,148 @@ const ShareDialogCozyToCozy = ({
   showShareByEmail,
   showShareByLink,
   showShareOnlyByLink,
-  showWhoHasAccess
+  showWhoHasAccess,
+  hasTwoStepsConfirmation = false,
+  twoStepsConfirmationMethods = {}
 }) => {
   const { t } = useI18n()
+
+  const [status, setStatus] = React.useState('loading')
+  const [recipientsToBeConfirmed, setRecipientsToBeConfirmed] = useState([])
+  const [recipientConfirmationData, setRecipientConfirmationData] = useState(
+    undefined
+  )
+
+  const {
+    confirmRecipient,
+    rejectRecipient,
+    recipientConfirmationDialogContent: ConfirmationDialogContent
+  } = twoStepsConfirmationMethods
+
+  const mountedRef = React.useRef(false)
+  useEffect(() => {
+    mountedRef.current = true
+
+    return () => {
+      mountedRef.current = false
+    }
+  })
+
+  const getRecipientsToBeConfirmed = useCallback(() => {
+    if (
+      hasTwoStepsConfirmation &&
+      twoStepsConfirmationMethods?.getRecipientsToBeConfirmed
```

Comment:
It seems that our babel config doesn't support this stuff by default, no? https://github.com/cozy/cozy-libs/blob/master/packages/babel-preset-cozy-app/index.js 

How it is transpiled ATM?

Reply:
It seems to be handled correctly, here is the `dist` output : 

```js
    if (hasTwoStepsConfirmation && (twoStepsConfirmationMethods === null || twoStepsConfirmationMethods === void 0 ? void 0 : twoStepsConfirmationMethods.getRecipientsToBeConfirmed)) {
```

Reply:
🤷‍♂️